lib/Tool/Session.php

```diff
@@ -29,8 +30,34 @@ final class Session
      */
     private static $handler;
 
+    /**
+     * @desc This is for forward compatibility, the custom Session implementation is not being used anymore in Pimcore 11.
+     * @desc For forward compatibility, you can use this class and pass the SessionInterface from the request, in Pimcore 10.6, the Admin Session will be used instead
+     *
+     * @param SessionInterface $session Session being used is the Admin Session, you can safely pass nothing here
+     *
+     * @param callable(AttributeBagInterface, SessionInterface):mixed $func
+     *
+     */
+    public static function useBag(SessionInterface $session, callable $func, string $namespace = 'pimcore_admin'): mixed
+    {
+        return static::useSession($func, $namespace);
+    }
+
+    public static function getSessionBag(
+        SessionInterface $session,
+        string $namespace = 'pimcore_admin'
+    ): ?AttributeBagInterface {
+        return static::get($namespace);
+    }
+
+    /**
+     * @deprecated
+     */
     public static function getHandler(): AdminSessionHandlerInterface
     {
+        trigger_deprecation('pimcore/pimcore', '10.6', 'Usage of \Pimcore\Tool\Session::getHandler tag is deprecated since version 10.6 and will be removed in Pimcore 11. No alternative given.');
```

Comment:
```suggestion
        trigger_deprecation('pimcore/pimcore', '10.6', sprintf('Usage of method %s  is deprecated since version 10.6 and will be removed in Pimcore 11. No alternative given.', __METHOD__));
```